handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+    Self signed certificates should be used for testing purposes only.
+
+Refer, to the [MongoDB manual](https://docs.mongodb.com/manual/appendix/security/appendixA-openssl-ca/) for steps to create and run `mongod` with a self-signed certificate.
+
+!!! note "Running MongoDB without Client certificate validation"
+
+    Start `mongod` with command line arguments:
+    ```bash
+    mongod --tlsMode requireTLS --tlscertificateKeyFile /etc/ssl/test-server1.pem --port 27017 --dbpath /var/lib/mongo --bind_ip_all
+    ```
+
+    Start `mongod` with a Mongo Configuration file:
+    ```yaml
+    net:
+        port: 27017
+        bindIp: 0.0.0.0
+        tls:
+            mode: requireTLS
+            certificateKeyFile: /etc/ssl/test-server1.pem
+    systemLog:
+        destination: file
+        logAppend: true
+        path: /var/log/mongodb/mongod.log
+    storage:
+        dbPath: /var/lib/mongo
+        journal:
+            enabled: true
+    processManagement:
+        fork: true  #fork and run in background
+        pidFilePath: /var/run/mongodb/mongod.pid  # location of pidfile
+        timeZoneInfo: /usr/share/zoneinfo
+    ```
+
+#### Installing MongoDB Self-Signed certificates
+
+If you are using a self signed certificate on your `mongod` instance you must install the certificate authority (CA) certificate and intermediary certificate into the SystemLink app server.
+
+1. Copy the CA and intermediary `.crt` files to you SystemLink app server.
+
+2. Double click on the CA certificate and click **Install certificate...**.
+
+    !!! note
+        If you are following the [MongoDB manual appendix](https://docs.mongodb.com/manual/appendix/security/appendixA-openssl-ca/) this file is named `mongodb-test-ca.crt`.
+
+3. In the **Store Location** field select the **Local Machine** radio button and click **Next**.
+
+4. Select the **Place all certificate in the following store** radio button.
+
+5. Click **Browse**, select **Trusted Root Certificate Authorities** and click **OK**
+
+6. Click **Next**, review the settings, and click **Finish** to install the certificate.
+
+7. Double click on the intermediary certificate and click **Install certificate...**.
+
+    !!! note
+        If you are following the [MongoDB manual appendix](https://docs.mongodb.com/manual/appendix/security/appendixA-openssl-ca/) this file is named `mongodb-test-ia.crt`.
+
+8. In the **Store Location** field select the **Local Machine** radio button and click **Next**.
+
+9. Select the **Place all certificate in the following store** radio button.
+
+10. Click **Browse**, select **Intermediate certificate Authorities** and click **OK**
+
+11. Click **Next**, review the settings, and click **Finish** to install the certificate.
+
+!!! note
+    Ensure you have checked the **Use Transport Layer Security (TLS) encryption** in the **NoSqlDatabase** section of the **SystemLink Server Configuration** application.
+
+## Deprecation of AMQP
```

Comment:
Following that email thread with Timo and Matt last week, I think this might make more sense[ in SPD](https://www.ni.com/documentation/en/systemlink/latest/setup/setting-up-systemlink-server/
)- **if** it's technically deprecated at all right now, which I don't think we came to a consensus on.

Reply:
I have added a requirement to https://ni.visualstudio.com/DevCentral/_workitems/edit/1206992 to update our documentation to state this in SPD as well.